ui/src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import { Container, Text } from "@chakra-ui/react"
+import React, { useState } from "react"
+import HeaderBar from "./components/HeaderBar"
+import StatusCardList from "./components/StatusCardList"
+import useFetchStatuses from "./hooks/useFetchStatuses"
+
+function App(): React.ReactElement {
+  const { stacks, error } = useFetchStatuses()
+  const [searchQuery, setSearchQuery] = useState("")
+
+  return (
+    <>
+      <Container maxW="container.lg" mt={4}>
+        <HeaderBar onQueryChange={query => setSearchQuery(query)} error={error !== null} />
```

Comment:
Some margin between the header bar and the status card list would like nicer imo

Reply:
Added margin